rskj-core/src/main/resources/config/testnet.conf

```diff
@@ -81,6 +81,11 @@ wallet {
     accounts = []
 }
 
+rpc {
```

Comment:
do we need this config in testnet?

Reply:
so can we remove it? not sure why you added it, the config is `testnet` specific